cmd/dogstatsd/main_windows.go

```diff
@@ -296,33 +316,22 @@ func importRegistryConfig() error {
 	} else {
 		log.Debug("proxy key not found, not setting proxy config")
 	}
-	if val, _, err = k.GetStringValue("site"); err == nil && val != "" {
-		overrides["site"] = val
-		log.Debugf("Setting site to %s", val)
-	}
-	if val, _, err = k.GetStringValue("dd_url"); err == nil && val != "" {
-		overrides["dd_url"] = val
-		log.Debugf("Setting dd_url to %s", val)
-	}
-	if val, _, err = k.GetStringValue("logs_dd_url"); err == nil && val != "" {
-		overrides["logs_config.logs_dd_url"] = val
-		log.Debugf("Setting logs_config.dd_url to %s", val)
-	}
-	if val, _, err = k.GetStringValue("process_dd_url"); err == nil && val != "" {
-		overrides["process_config.process_dd_url"] = val
-		log.Debugf("Setting process_config.process_dd_url to %s", val)
-	}
-	if val, _, err = k.GetStringValue("trace_dd_url"); err == nil && val != "" {
-		overrides["apm_config.apm_dd_url"] = val
-		log.Debugf("Setting apm_config.apm_dd_url to %s", val)
-	}
-	if val, _, err = k.GetStringValue("py_version"); err == nil && val != "" {
-		overrides["python_version"] = val
-		log.Debugf("Setting python version to %s", val)
+
+	for winRegKey, yamlSettingKey := range regSettings {
+		if val, _, err = k.GetStringValue(winRegKey); err == nil && val != "" {
+			log.Debugf("Setting %s to %s", yamlSettingKey, val)
+			overrides[yamlSettingKey] = val
+		}
```

Comment:
We are also making changes to (actually, removing) this code in https://github.com/DataDog/datadog-agent/pull/7204, so this will cause conflicts. If you don't want to wait for that PR to be ready (it still isn't), can I ask you to keep the changes to a minimum to ease the merge? (eg: don't refactor this to a loop).